MAUI/Maps/vector-layers/polygon-layer.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: Adding polygons in Maps control | Syncfusion
+description: Learn here all about adding the Polygons feature of Syncfusion .NET MAUI Maps (SfMaps) control to customize their appearances and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Polygons in .NET MAUI Maps (SfMaps)
+
+Polygon layer is a sublayer that renders a group of `MapPolygon` on `MapShapeLayer`. This section helps to learn about how to add the polygons and customize them.
```

Comment:
Polygon layer is a sublayer that renders a group of `MapPolygon` on the `MapShapeLayer.` This section helps to learn about how to add the polygons and customize them.

Reply:
Changed the content as mention above